### PR TITLE
Shows typescript errors in danger

### PR DESCRIPTION
**Pull request by orta**
Migrates all our `ci`-y commands into a `ci:` prefix in scripts. Adds one that pipes tic's output to a file and prints it to danger. This should happen in this PR.

### CODE REVIEW

**Comment by alloy** [comment]
Nice 👌 

Looks like you'll need to strip ANSI codes.

**Comment by ashfurrow** [comment]
What a good idea!

File: package.json [hunk 1]
```diff
@@ -13,14 +13,17 @@
     "bundle": "yarn generate-graphql-query-map && yarn generate-cities-objc && yarn generate-cities-cache && react-native bundle --platform=ios --dev=false --entry-file=index.ios.js --bundle-output Pod/Assets/Emission.js --sourcemap-output Pod/Assets/Emission.js.map --assets-dest Pod/Assets && rm -rf Pod/Assets/assets/node_modules",
     "bundle-for-native-ci": "yarn generate-graphql-query-map && react-native bundle --platform=ios --dev=false --entry-file=index.tests.ios.js --bundle-output Pod/Assets/Emission.js --sourcemap-output Pod/Assets/Emission.js.map --assets-dest Pod/Assets && rm -rf Pod/Assets/assets/node_modules",
     "bundle-with-storybooks": "yarn generate-graphql-query-map && react-native bundle --platform=ios --dev=true --entry-file=Example/Emission/index.ios.js --bundle-output  Pod/Assets/Emission.js --assets-dest Pod/Assets",
-    "type-check": "tsc --noEmit --pretty",
+    "type-check": "tsc",
```
Comment (orta) [inline package.json]: these are set in the tsconfig

**Comment by alloy** [comment]
Dope 👍

**Pull request merged**